### PR TITLE
Publish v1.6.0 of the v1 API OpenAPI specification

### PR DESCRIPTION
**Pull request by BrendanSChang**
### Your checklist for this pull request

Thank you for your contribution to the `next-dev-docs` repo. 

🚨Please review the [Pull request guidelines](../contributor-guide/contributor-guide.md#pull-request-guidelines) to this repository before submitting this PR.

- [ ] Your code builds clean without any errors or warnings
  
  ```bash
  yarn build
  ```

- [ ] You are using approved terminology

  Refer to the [style guide](../contributor-guide/style-guide.md) for help.

### Description 

Publish v1.6.0 of the v1 API OpenAPI specification. This removes "hosted_web" as an authenticator type and replaces it with "hosted_login".

### CODE REVIEW

**Comment by BrendanSChang** [comment]
@patriciamcphee a few questions on this:
- Are we still using this repository or should these changes be posted elsewhere (e.g. customer-docs)?
- Are there release notes for Secure Customer/is there some place I should record the changes from this release?

File: static/api/v1/openapi.yaml [hunk 1]
```diff
@@ -4793,6 +4795,28 @@ paths:
...
+                Success Hosted Web:
+                  value:
+                    id: 73731b7f-eb76-4143-9b4b-81a720385f5a
+                    realm_id: caf2ff640497591a
+                    tenant_id: 00011f1183c67b69
+                    display_name: Pet Authenticator Configuration
+                    config:
+                      type: hosted_web
+                      trusted_origins:
+                        - 'http://localhost:8092'
```
Comment (Will5) [inline static/api/v1/openapi.yaml:4819]: I thought we deprecated Hosted Web. Did we miss a location?

Comment (BrendanSChang) [reply static/api/v1/openapi.yaml:4819]: Oh, if they're all supposed to be removed, then yes. They are still a few artifacts under examples for CreateAuthenticatorConfig and UpdateAuthenticatorConfig. And the resource for the hosted login config is still called "Hosted Web Authenticator".

Comment (Will5) [reply static/api/v1/openapi.yaml:4819]: @ismailmustafa, can you advise here? We probably don't want any references to "hosted_web" anymore, correct?

**Comment by BrendanSChang** [comment]
There are some remaining artifacts to update, so closing this PR in favor of the subsequent which will include all of the updates.

**Pull request closed without merging**